content/chainguard/chainguard-images/faq.md

```diff
@@ -14,35 +14,74 @@ weight: 600
 toc: true
 ---
 
-### Which Linux distribution is used as base for Chainguard Images?
+Learn answers to your questions about [Chainguard Images](https://www.chainguard.dev/chainguard-images?utm_source=docs).
+
+## Which Linux distribution is used as base for Chainguard Images?
 Chainguard images are based on [Wolfi](/open-source/wolfi/), a Linux _undistro_ we built specifically to address software supply chain security issues.
 We do have some images with Alpine-based variants in order to support musl or unusual architectures.
 
-### How do Chainguard Images relate to the Google Distroless Images?
+## How do Chainguard Images relate to the Google Distroless Images?
 The [Google distroless](https://github.com/GoogleContainerTools/distroless) images follow a similar
 philosophy to many of our images: they are minimal images that don't include package managers or
 shells. The main difference is in the implementation. The Google distroless images are built with
 [Bazel](https://bazel.build) and based on the Debian distribution, whereas Chainguard Images are
 built with [apko](/open-source/apko) based on the [Wolfi](/open-source/wolfi) or Alpine
 distributions. We believe our approach is more maintainable and extensible.
 
-### What is an "undistro"?
+## What is an "undistro"?
 We call Wolfi an undistro because unlike a typical Linux distribution, Wolfi is a stripped-down distribution designed for the cloud-native era. Most notably, we don’t include a Linux kernel, instead relying on the environment (such as the container runtime) to provide this.
 
-### Which images are available?
-You can check which images are already available at our [Images Catalog](https://edu.chainguard.dev/chainguard/chainguard-images/reference/), through the [Chainguardy Registry](/chainguard/chainguard-images/registry/overview/), or in our [GitHub Repository](https://github.com/chainguard-images).
+## Which images are available?
+
+Chainguard Images currently support more than 100+ images, which are segmented in [three catalog tiers](#what-are-the-different-catalog-tiers-of-chainguard-images), which you can read about in the [next question](#what-are-the-different-catalog-tiers-of-chainguard-images).
```

Comment:
```suggestion
Chainguard currently support more than 100 Images, which are segmented in [three catalog tiers](#what-are-the-different-catalog-tiers-of-chainguard-images), which you can read about in the [next question](#what-are-the-different-catalog-tiers-of-chainguard-images).
```
"more than 100+" felt redundant. Also, not sure about splitting the product name like this, feel free to revert it, I just thought it read a little funny to have the word "images" in the same clause

Reply:
I rephrased this :)